packages/clerk-js/src/core/resources/SignUp.ts

```diff
@@ -487,11 +488,34 @@ class SignUpFuture implements SignUpFutureResource {
     return this.resource.unverifiedFields;
   }
 
+  private async getCaptchaToken(): Promise<{ captchaToken?: string; captchaWidgetType?: CaptchaWidgetType }> {
+    const captchaChallenge = new CaptchaChallenge(SignUp.clerk);
+    const response = await captchaChallenge.managedOrInvisible({ action: 'signup' });
+    if (!response) {
+      throw new Error('Captcha challenge failed');
+    }
+
+    const { captchaError, captchaToken, captchaWidgetType } = response;
+    if (captchaError) {
+      throw new Error(captchaError);
```

Comment:
❓ do we know if `captchaError` is already an error type? Wondering if it's necessary to wrap it in `new Error()` again here 👀